### ISSUE TITLE
Error installing on Windows

### ISSUE THREAD

**Issue by abhikco** [audreyt/node-webworker-threads#78]
On Windows 7, Node v0.12.2, "npm install webworker-threads" leads to the following errors:

```
D:\scheduler_client>npm install webworker-threads
npm WARN package.json scheduler_client@0.0.1 No repository field.
-
> webworker-threads@0.6.1 install D:\scheduler_client\node_modules\webworker-threads
> node-gyp rebuild


D:\scheduler_client\node_modules\webworker-threads>if not defined npm_config_node_gyp (node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\\..\..\node_modules\node-gyp\bin\node-gyp.js" rebuild )  else (rebuild)
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  WebWorkerThreads.cc
C:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\include\xlocale(323): warning C4530: C++ exception handler used, but unwind semantics are not enabled. Specify /EHsc [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
d:\scheduler_client\node_modules\webworker-threads\src\bson.cc(891): warning C4267: 'argument' : conversion from 'size_t' to 'uint32_t', possible loss of data [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
d:\scheduler_client\node_modules\webworker-threads\src\jslib.cc(79): warning C4344: behavior change: use of explicit template arguments results in call to 'Nan::MaybeLocal<T> Nan::New<v8::String,char*>(A0)' [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
          with
          [
              T=v8::String,
              A0=char *
          ]
          but the regular function 'Nan::MaybeLocal<T> Nan::New(const char *)' is a better match
          with
          [
              T=v8::String
          ]
          if you expect 'Nan::MaybeLocal<T> Nan::New(const char *)' to be called then you need to make it an explicit specialization
          with
          [
              T=v8::String
          ]
..\src\WebWorkerThreads.cc(330): error C2065: 'MaybeLocal' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(330): error C2275: 'v8::Script' : illegal use of this type as an expression [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
          C:\Users\i306608\.node-gyp\0.12.2\deps\v8\include\v8.h(983) : see declaration of 'v8::Script'
..\src\WebWorkerThreads.cc(330): error C2065: 'script' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(333): error C2065: 'MaybeLocal' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(333): error C2275: 'v8::Value' : illegal use of this type as an expression [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
          C:\Users\i306608\.node-gyp\0.12.2\deps\v8\include\v8.h(1345) : see declaration of 'v8::Value'
..\src\WebWorkerThreads.cc(333): error C2065: 'result' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(333): error C2065: 'script' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(333): error C2228: left of '.ToLocalChecked' must have class/struct/union [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
          type is ''unknown-type''
..\src\WebWorkerThreads.cc(334): error C2065: 'result' : undeclared identifier [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
..\src\WebWorkerThreads.cc(334): error C2228: left of '.ToLocalChecked' must have class/struct/union [D:\scheduler_client\node_modules\webworker-threads\build\WebWorkerThreads.vcxproj]
          type is ''unknown-type''
gyp ERR! build error
gyp ERR! stack Error: `C:\WINDOWS\Microsoft.NET\Framework\v4.0.30319\msbuild.exe` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:269:23)
gyp ERR! stack     at ChildProcess.emit (events.js:110:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:1074:12)
gyp ERR! System Windows_NT 6.1.7601
gyp ERR! command "node" "C:\\Program Files\\nodejs\\node_modules\\npm\\node_modules\\node-gyp\\bin\\node-gyp.js" "rebuild"
gyp ERR! cwd D:\scheduler_client\node_modules\webworker-threads
gyp ERR! node -v v0.12.2
gyp ERR! node-gyp -v v1.0.3
gyp ERR! not ok
npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Program Files\\nodejs\\\\node.exe" "C:\\Program Files\\nodejs\\node_modules\\npm\\bin\\npm-cli.js" "install" "webworker-threads"
npm ERR! node v0.12.2
npm ERR! npm  v2.7.4
npm ERR! code ELIFECYCLE

npm ERR! webworker-threads@0.6.1 install: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the webworker-threads@0.6.1 install script 'node-gyp rebuild'.
npm ERR! This is most likely a problem with the webworker-threads package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-gyp rebuild
npm ERR! You can get their info via:
npm ERR!     npm owner ls webworker-threads
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     D:\scheduler_client\npm-debug.log
```
Regards
Abhik

**Comment by audreyt** [comment]
Hi, this should be fixed as of 0.6.2, please confirm. Thanks for the report!

**Comment by abhikco** [comment]
Works like a charm.... Thanks a lot !!
